plugins/CoreHome/templates/_dataTable.twig

```diff
@@ -76,6 +76,14 @@
     </div>
 </div>
 
+{% if notifications is not empty and notifications|length %}
```

Comment:
when multiple reports are rendered on one page and they all trigger a notification, will they all be added multiple times?

Reply:
Depends on whether they use an ID or not. With an ID that doesn't change based on report, only one shows up. If no IDs are used, then I think multiple ones show up.

In this PR, since it uses an ID for the notification, there's only ever one.